js-pkg/.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
cool, til. Does nvm pick this up automatically when you run `node` under this directory?

Reply:
unfortunately, no, you have to do an `nvm use` and it'll pick up the first nvmrc as it goes up the filesystem